### DISCUSSION TITLE
pimcore 10 validation on autosave

### DISCUSSION THREAD

**Discussion by shubhangi92** [pimcore/pimcore#10213] [General]
Hi,
 
Pimcore 10 auto save is througing validation errors for all fields even make change on single field .ie. if you select date it gives validation error for all other fields on change .

Thanks
Shubhangi

**Comment by dvesh3** [comment]
@shubhangi92 Auto save calls "Save a version" internally, which never validates the required fields. Validation only happens on "publish and save". Could you please share reproducible steps? thanks!

**Reply by shubhangi92** [reply]
I have added some validations checks on postUpdate event , now when i make any change on particular field  i.e i drag drop relations or date change it automatically calls postUpdate event and show validation error messages. while it should only work when i click on save button

**Reply by shubhangi92** [reply]
i wanted to apply validation checks on save click if any alternate solution  please suggest

**Reply by dvesh3** [reply]
you check for `isAutoSave` argument and return control in postUpdate event:
```
 if ($e->hasArgument('isAutoSave') && $e->getArgument('isAutoSave')) {
      return;
  }
```

**Reply by shubhangi92** [reply]
Thanks It works!!
